BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditItemProcessor.swift

```diff
@@ -175,6 +176,11 @@ final class AddEditItemProcessor: StateProcessor<// swiftlint:disable:this type_
             state.owner = newValue
         case let .passwordChanged(newValue):
             state.loginState.password = newValue
+        case .removePasskeyPressed:
+            state.loginState.fido2Credentials = []
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
+                UIAccessibility.post(notification: .announcement, argument: Localizations.passkeyRemoved)
+            }
```

Comment:
🤔  @matt-livefront @KatherineInCode Do you know if there is away to mock this without wrapping it in some service we can inject?

Reply:
I don't think so, it would need a thin wrapper to be able to mock it.

Reply:
@LRNcardozoWDF alright let's just merge this and then we can think the best way for accessibility testing. I'll create a tech-debt ticket so we can address it.